cartography/intel/github/users.py

```diff
@@ -84,27 +91,37 @@ def load_organization_users(
     u.company = user.node.company,
     u.lastupdated = $UpdateTag
 
-    MERGE (u)-[r:MEMBER_OF]->(org)
+    MERGE (u)<-[r:MEMBER_OF]-(org)
```

Comment:
same as above. let's keep the relationships same as upstream. unless absolutely necessary